Author: Nathan Phillip Brink <binki@ohnopub.net>
Purpose: To make sure that LDFLAGS are actually used in the correct places in atheme. Adds LDFLAGS to buildsys.module.mk, somewhat similar to the ``${PLUGIN} ${PLUGIN_NOINST}:'' rule for shared objects in buildsys.mk.in. Also fixes a loss of LDFLAGS in configure.ac if the platform is ``Tru64''.

diff -r 40ec7097b8e3 buildsys.module.mk
--- a/buildsys.module.mk	Tue Jun 29 01:36:51 2010 -0500
+++ b/buildsys.module.mk	Tue Aug 31 22:03:14 2010 -0400
@@ -7,7 +7,7 @@
 
 .c$(PLUGIN_SUFFIX):
 	${COMPILE_STATUS}
-	if ${CC} ${PLUGIN_CFLAGS} ${PLUGIN_LDFLAGS} ${CFLAGS} ${CPPFLAGS} ${LIBS} -o $@ $<; then \
+	if ${CC} ${CFLAGS} ${PLUGIN_CFLAGS} ${CPPFLAGS} ${PLUGIN_LDFLAGS} ${LDFLAGS} -o $@ $< ${LIBS}; then \
 		${COMPILE_OK}; \
 	else \
 		${COMPILE_FAILED}; \
diff -r 40ec7097b8e3 configure.ac
--- a/configure.ac	Tue Jun 29 01:36:51 2010 -0500
+++ b/configure.ac	Tue Aug 31 22:03:14 2010 -0400
@@ -339,7 +339,7 @@
 	if test "$Tru" = yes -a "$CC" != gcc; then
 		AC_MSG_RESULT([Tru64: -shared -expect_unresolved '*'])
 		PICFLAGS="-shared -expect_unresolved '*' "
-		LDFLAGS="-call_shared"
+		LDFLAGS="${hold_ldflags} -call_shared"
 	fi
 
 	if test "$CC" = tcc -a "$TenDRA" = no; then
